packages/sprite/src/Sprite.ts

```diff
@@ -271,18 +309,18 @@ export class Sprite extends Container
         {
             // if the sprite is trimmed and is not a tilingsprite then we need to add the extra
             // space before transforming the sprite coords.
-            w1 = trim.x - (anchor._x * orig.width);
+            w1 = trim.x - (anchor.x * orig.width);
```

Comment:
See the problem? @finscn wont be happy that we removed his "optimizations". That's a kind of hotspot and it uses extra  property calls. LEts confirm that @GoodBoyDigital is ok with that.

Anchor is always ObservablePoint so maybe we need to make `_x` and `_y` visible, its not the only place we use them.

Reply:
I've made `_x` and `_y` internal and reverted these changes